draft-ietf-taps-interface.md

```diff
@@ -948,6 +948,22 @@ is usually combined with congestion control in protocol implementations,
 rendering "reliable but not congestion controlled" a request that is unlikely to
 succeed.
 
+## Keep alive {#keep-alive}
+
+Name:
+: keepAlive
+
+Type:
+: Preference
+
+Default:
+: Ignore
+
+This property specifies whether the application would like the Connection to send
+keep-alive packets or not. Note that if a Connection determines that keep-alive
+packets are being sent, the applicaton should itself avoid generating additional keep alive
+message {{!BCP145}}. Note that when supported, the system also needs to
```

Comment:
Where does BCP145 actually say that? Usually if the application sends keeps-lives with a higher rate that just mean the transport would never do as there is never a large enough idle time. So that's no problem. BCP145 says "Keep-alive messages are NOT RECOMMENDED for general use." however this is specific recommendation for UDP as time-out usually have to be low(er) for UDP (than TCP).

Reply:
I was trying to find a REF that said don't waste bandwidth on excessive keep-alives. BCP145 probably wasn't the best, I'm removing that - because it's not the focus at all - so do we need a REF?

Reply:
Don't waste bandwidth is a good advise and BCP145 might be a good reference, however, it not relevant on switch layer you waste the bandwidth :-)